docs/walkthrough-1.6.md

```diff
@@ -3,146 +3,18 @@
 This document outlines the basic features of the service catalog by walking
 through a short demo.
 
-## Step 0 - Prerequisites
+This document contains instructions for a self-guided demo of the Service 
+Catalog for Kubernetes clusters running version 1.6. Since Service
+Catalog only officially supports versions 1.7 and later, these instructions are 
+deprecated and may be removed at any time.
 
-### Starting Kubernetes with DNS
+If you are running a Kubernetes cluster running version 1.7 or later, please 
+see [walkthrough-1.7.md](./walkthrough-1.7.md).
 
-You *must* have a Kubernetes cluster with cluster DNS enabled. We can't list
-instructions here for enabling cluster DNS for all Kubernetes cluster
-installations, but here are a few notes:
+This document assumes that you've installed Service Catalog onto your cluster.
+If you haven't, please see [install-1.6.md](./install-1.6.md).
```

Comment:
clean up the link text here and above to be something like "the installation instructions for v1.X"

Reply:
done